File: Lista_2/Exercicios_CASA-lista-2/EX7-Cursos-c/EX7-c.c
#include <stdio.h>

int main() {
    
    int num;
    scanf("%d", &num);
    char *meses[12] = {"Engenharia","Edificações","Sistemas Elétricos","Turismo","Análise de Sistemas"};
    if (1 <= num && num <= 5){
        printf("%s\n", meses[num-1]);
    }
    else{
        printf("Numero Invalido\n");
    }
    
    return 0;
}

File: _IFSP/LP1/Lista_2/Exercicios_CASA-lista-2/EX7-Cursos-c/EX7-c.py
num = int(input())
omes = "numero invalido"
meses=("Engenharia","Edificações","Sistemas Elétricos","Turismo","Análise de Sistemas")
if 1 <= num <= len(meses):
    omes = meses[num - 1]
print(omes)